addons/mail_bot/views/res_users_views.xml

```diff
@@ -1,19 +1,5 @@
 <?xml version="1.0"?>
 <odoo><data>
-    <!-- Update Preferences form !-->
-    <record id="res_users_view_form_preferences" model="ir.ui.view">
-        <field name="name">res.users.view.form.preferences.mail_bot</field>
-        <field name="model">res.users</field>
-        <field name="inherit_id" ref="mail.view_users_form_simple_modif_mail"/>
-        <field name="arch" type="xml">
-            <data>
-                <field name="notification_type" position="after">
-                    <field name="odoobot_state" readonly="0" groups="base.group_no_one"/>
-                </field>
-            </data>
-        </field>
-    </record>
```

Comment:
revert this diff, this part stays here, no need to move it below on the file.

Just `view_users_form_inherit_mail_notification_alert` can be removed